cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
Is this file needed?

Reply:
It's not required, it was just generated by the Cypress setup, and is [used for extending Cypress](https://docs.cypress.io/api/cypress-api/custom-commands.html#Syntax). Figured I'd go ahead and leave it, since it's tucked away in the `cypress` folder, and when someone needs it, it'll be there.